jsonchecker/__init__.py

```diff
@@ -42,7 +42,8 @@ def check_directory(self, directory):
         else:
             files = [directory]
         for fname in files:
-            if fname.startswith('.'):
+            first_directory = fname.split(os.path.sep)[0]
```

Comment:
this solution also skips files if they are requested like
```
$ jsonchecker ./good.json 
```

I believe this is intended to prevent looking at dot-files , because `os.listdir` doesnt return entries for `.` or `..`, and dot-directories (like digging around in `.git` or `.hg` which will be a waste of effort)  Your current solution doesnt prevent that.